src/lang/es-MX.json

```diff
@@ -112,11 +112,13 @@
 	"app.thank-you-heading": "¡Gracias!",
 	"app.thank-you-text": "Está todo listo.",
 	"app.go-to-profile": "Mi perfil",
+	"error.email-format": "Your email should have a format similar to 'john@gmail.com'",
```

Comment:
@trigal2012 these are in English. shouldn't they be in Spanish?

Reply:
@FJKhan yes, there are some that are not yet translated. I put them there as placeholders. I'll have all that sorted out before the final PR. This is the basics of the signup flow, which is still very much in progress.

The alphabetized PR I mentioned on the call today is #182, https://github.com/asylum-connect/asylumconnect-catalog/pull/182

Reply:
I would add WIP: at the begging of the title of the PR. It will prevent status checks from running and let us know we should not review or merge this.

Reply:
@Alfredo-Moreira thanks for that info. I've added WIP to the title.